### PR TITLE
Initial diffraction release notes for v6.8

### PR DESCRIPTION
**Pull request by RichardWaiteSTFC**
**Description of work**

Initial diffraction release notes for v6.8

**To test:**
(1) Build docs
(2) Check links and formatting in html

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Comment by sf1919** [comment]
This needs to be on release-next

File: docs/source/release/v6.8.0/diffraction.rst [hunk 1]
```diff
@@ -10,34 +10,57 @@ Powder Diffraction
...
+- :ref:`PDCalibration <algm-PDCalibration>` has two new parameters, ``StartWorkspaceIndex`` and ``StopWorkspaceIndex``, which limit which spectra are calibrated
+- :ref:`PowderReduceP2D <algm-PowderReduceP2D>` has been adapted to work with SNAP measuring data.
+- :ref:`AlignAndFocusPowderFromFiles <algm-AlignAndFocusPowderFromFiles>`, when it calls :ref:`LoadDiffCal <algm-LoadDiffCal>`, it will pass through the `TMin` parameter. In the case of non-zero `TZERO` values existing in the calibration constant, the implementation of current PR will suppress the warning message about minimum TOF value shfit.
 
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Powder/Bugfixes
+- Fixed bug in absolute unit normalisation in ISIS powder reduction where the vanadium workspace shape was used instead of the sample.
+- Raise errors in absolute normalisation due to lack of sample/vanadium shape and material in ISIS powder reduction
+- Raise an error in POLARIS (ISIS) powder reduction script if user tries to creta a vanadium workspace without correcting for absorption and multiple scattering.
```
Comment (ConorMFinn) [inline docs/source/release/v6.8.0/diffraction.rst:27]:
```suggestion
- Raise an error in POLARIS (ISIS) powder reduction script if user tries to create a vanadium workspace without correcting for absorption and multiple scattering.
```

File: docs/source/release/v6.8.0/diffraction.rst [hunk 2]
```diff
@@ -10,34 +10,57 @@ Powder Diffraction
...
+- Expose the OffsetData parameter in the Powder Diffraction Reduction gui
+- :ref:`PDCalibration <algm-PDCalibration>` has two new parameters, ``StartWorkspaceIndex`` and ``StopWorkspaceIndex``, which limit which spectra are calibrated
+- :ref:`PowderReduceP2D <algm-PowderReduceP2D>` has been adapted to work with SNAP measuring data.
+- :ref:`AlignAndFocusPowderFromFiles <algm-AlignAndFocusPowderFromFiles>`, when it calls :ref:`LoadDiffCal <algm-LoadDiffCal>`, it will pass through the `TMin` parameter. In the case of non-zero `TZERO` values existing in the calibration constant, the implementation of current PR will suppress the warning message about minimum TOF value shfit.
 
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Powder/Bugfixes
+- Fixed bug in absolute unit normalisation in ISIS powder reduction where the vanadium workspace shape was used instead of the sample.
+- Raise errors in absolute normalisation due to lack of sample/vanadium shape and material in ISIS powder reduction
```
Comment (ConorMFinn) [inline docs/source/release/v6.8.0/diffraction.rst:26]: This reads a bit strangely
```suggestion
- When performing an ISIS powder reduction, errors are raised in absolute normalisation when sample/vanadium shape and material are not provided.
```

File: docs/source/release/v6.8.0/diffraction.rst [hunk 3]
```diff
@@ -10,34 +10,57 @@ Powder Diffraction
...
 -----------------------
 
 New features
 ############
-.. amalgamate:: Diffraction/Engineering/New_features
-
-Bugfixes
-############
-.. amalgamate:: Diffraction/Engineering/Bugfixes
+- Added new SNAPRed menu item for Diffraction interfaces
```
Comment (ConorMFinn) [inline docs/source/release/v6.8.0/diffraction.rst:38]:
```suggestion
- Added new ``SNAPRed`` menu item for Diffraction interfaces
```

File: docs/source/release/v6.8.0/diffraction.rst [hunk 4]
```diff
@@ -10,34 +10,57 @@ Powder Diffraction
...
 
 New features
 ############
-.. amalgamate:: Diffraction/Single_Crystal/New_features
+- :ref:`LoadWANDSCD <algm-LoadWANDSCD>` has new keyword argument to apply or not apply goniometer tilts
+- Update fitting coefficients for :ref:`BackToBackExponential <func-BackToBackExponential>` in SXD parameters xml file.
+- Add parameter ``MinIntensOverSigma`` in :ref:`SaveReflections <algm-SaveReflections>` to filter peaks to export by Intensity/Sigma
+- Support saving :ref:`LeanElasticPeaksWorkspace <LeanElasticPeaksWorkspace>` with GSAS format in :ref:`SaveReflections <algm-SaveReflections>` (calls :ref:`SaveHKLCW <algm-SaveHKLCW>`)
+- New parameter in :ref:`SaveIsawUB <algm-SaveIsawUB>` to rotate the UB by the goniometer matrix.
+- New algorithm to load gaussian cube files for volumetric data into a 3D :ref:`MDHistoWorkspace <MDHistoWorkspace>`
```
Comment (ConorMFinn) [inline docs/source/release/v6.8.0/diffraction.rst:51]: Can we have a link to this new algorithm's docs?

**Review by cailafinn — CHANGES_REQUESTED** [review]
A few small issues/questions

**Review by cailafinn — APPROVED** [review]

**Pull request merged**